src/media/playback/playback_sensor.h

```diff
@@ -2,10 +2,10 @@
 // Copyright(c) 2017 Intel Corporation. All Rights Reserved.
 
 #pragma once
-#include <core/roi.h>
-#include <core/extension.h>
-#include <core/serialization.h>
-#include "core/streaming.h"
+#include "../../core/roi.h"
```

Comment:
What did we moved to this path's? didn't we say we don't want to manually "walk" in the filesystem?

Reply:
`src/` includes can be used from outside src/. as we know happens from unit-tests. But in unit-tests, src/ is no longer part of the include directories.
Headers inside src/ know where they are and so can relate to one-another relatively, regardless of include directories. This independence is what I wanted to achieve. Eventually, even the realsense2 module itself will not have src/ as one of its includes.